src/js/components/Widgets/PositionSupportOpposeSnippet.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { Component, PropTypes } from "react";
+
+export default class PositionSupportOpposeSnippet extends Component {
+  static propTypes = {
+    candidate_display_name: PropTypes.string,
+    is_on_candidate_page: PropTypes.bool,
+    is_support: PropTypes.bool.isRequired,
+    is_oppose: PropTypes.bool.isRequired,
+    more_info_url: PropTypes.string,
+    speaker_display_name: PropTypes.string,
+    statement_text: PropTypes.string
+  };
+
+  render () {
+    var src;
+    var className;
+    var alt;
+    var positionLabel;
+    var isSupportedBy;
+    if (this.props.is_support){
+      src = "/img/global/icons/thumbs-up-color-icon.svg";
+      className = "explicit-position__icon";
+      alt = "Supports";
+      positionLabel = "Supports";
+      isSupportedBy = "is Supported by"
+    } else if (this.props.is_oppose) {
+      src = "/img/global/icons/thumbs-down-color-icon.svg";
+      className = "explicit-position__icon";
+      alt = "Opposes";
+      positionLabel = "Opposes";
+      isSupportedBy = "is Opposed by"
+    } else {
+      src = "/img/global/icons/mixed-rating-icon.svg";
+      className = "position-rating__icon position-rating__icon--mixed";
+      alt = "Neutral Rating";
+      positionLabel = "Neutral Rating of";
+      isSupportedBy = "is viewed Neutrally by"
```

Comment:
In what context would this get used? I don't remember discussing neutral positions among what I've been referring to as explicit positions (support vs oppose).